### PR TITLE
Test issue 223

### PR DESCRIPTION
**Pull request by Jan-E**
checkout 7.1.3 and test php-cgi -m

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -23,8 +23,8 @@ build_script:
 
     C:\projects\php-sdk\bin\phpsdk_setvars.bat
 
-    git clone --depth=1 https://github.com/php/php-src C:\projects\php-src
-
+    git clone https://github.com/php/php-src C:\projects\php-src
+
```
Comment (Stricted) [inline appveyor.yml]: you should avoid these identations, appvoyer dont like them...

**Comment by Jan-E** [comment]
Ouch. If I do exactly the same on my Windows 7 Pro this is the result:

apcu3

The same happens if I download the master.zip from https://ci.appveyor.com/project/krakjoe/apcu/build/master.46/job/c92t5i5dh4cv3f24/artifacts and use that to load php_apcu.dll

I will reboot my laptop with Windows 10 Pro to see if it is the same under that OS. https://github.com/krakjoe/apcu/issues/223 is a really nasty one.

**Comment by Jan-E** [comment]
It is OS-specific!
```
Microsoft Windows [Version 10.0.14393]
(c) 2016 Microsoft Corporation. All rights reserved.

C:\Windows\system32>d:

D:\>cd \zip\apcu_master

D:\zip\apcu_master>php -m
[PHP Modules]
apcu
Core
date
pcre
Reflection
SPL
standard

[Zend Modules]


D:\zip\apcu_master>php-cgi -m
[PHP Modules]
apcu
cgi-fcgi
Core
date
pcre
Reflection
SPL
standard

[Zend Modules]


D:\zip\apcu_master>
```

**Comment by Jan-E** [comment]
@Stricted I cannot reproduce the crash on Appveyor. Is there a way to build something with Visual Studio 2015 (vc14) and run it on a Windows Server 2008 or Windows 7 instance (both with the VC14 redistributable)?

**Comment by Stricted** [comment]
@Jan-E afaik you can only choose the visual studio version but not the os version :/

**Comment by Jan-E** [comment]
Closing this PR because it seems to be not possible to reproduce https://github.com/krakjoe/apcu/issues/223 on appveyor.

**Pull request closed without merging**